bundles/org.openhab.binding.opengarage/README.md

```diff
@@ -0,0 +1,54 @@
+# OpenGarage Binding
+
+The OpenGarage binding allows you to control an OpenGarage controller (https://opensprinkler.com/product/opengarage/) using OpenHAB
+
+
+## Supported Things
+
+Opengarage controllers from https://opensprinkler.com/product/opengarage/ are supported.
+
+## Discovery
+
+Auto-discover is not currently supported. You need to manually add a new item using it's IP address.
```

Comment:
Whole file: please start every sentence on a new line. That makes changes easier to review later.

Reply:
I believe this is addressed in 4714fc9b7d